.github/workflows/test.yml

```diff
@@ -46,6 +46,7 @@ jobs:
           LANTERN_EXPORTFILE_WAIT: 300
           LANTERN_EXPORT_NUMWORKERS: 50
           LANTERN_EXPORT_DURATION: 120
+          LANTERN_PRUNING_THRESHOLD: 43800
```

Comment:
This new environment variable should be added to the `env.sample` file and should also be defined in the endpointmanager `README.md`!